mock/partnership_registration_v1.json

```diff
@@ -71,31 +71,31 @@
         "surname": "FATHER",
         "occupation": "Father",
         "retired": true,
-        "designation": null,
+        "designation": "Undesignated",
```

Comment:
I think desgnation's might be specific things. Though I can't think what. Might be worth checking the test data.

Reply:
They could well be set things, but the fields are just text, so all I'm really worried about is the value being included.

Reply:
The problem is that it makes the test data less realistic overall.

Reply:
Fixed